website/docs/main/Badge.md

```diff
@@ -0,0 +1,114 @@
+---
+id: badge
+
+title: Badge
+
+slug: /badge
+---
+
+import Usage from './usage/Badge/Badge.md'
+
+Badges are small components typically used to communicate a numerical value or indicate the status of an item to the user.
+
+## Usage
+
+<Usage />
+
+---
+
+## Props
+
+### Badge
+
+- [Component](#component)
+- [badgeStyle](#badgestyle)
+- [containerStyle](#containerstyle)
+- [onPress](#onpress)
+- [status](#status)
+- [textProps](#textprops)
+- [textStyle](#textstyle)
+- [value](#value)
+
+## Reference
+
+### Badge
+
+#### Component
+
+Custom component to replace the badge outer component.
+
+| Type            | Default                           |
+| --------------- | --------------------------------- |
+| React Component | onPress ? TouchableOpacity : View |
+
+---
+
+#### badgeStyle
+
+Additional styling for badge (background) view component.
+
+| Type               | Default |
+| ------------------ | ------- |
+| View style(Object) | None    |
+
+---
+
+#### containerStyle
+
+Style for the container.
+
+| Type               | Default |
+| ------------------ | ------- |
+| View style(Object) | None    |
+
+---
+
+#### onPress
+
+Function called when pressed on the badge.
+
+| Type                    | Default |
+| ----------------------- | ------- |
+| (...args: any[]) => any | None    |
```

Comment:
Docs shouldn't expose internal implementation.

Reply:
What should be the appropriate type for this?

Reply:
`(...args) => void`
Return type should be void